XMonad/Hooks/EwmhDesktops.hs

```diff
@@ -548,7 +548,7 @@ fullscreenEventHook'
     when (action == add || (action == toggle && not isFull)) $ do
```

Comment:
Hm, do we perhaps want to make the same change here to prevent windows from being fullscreened multiple times if a rogue app makes multiple add requests?

Reply:
I left that one alone because it seems to be idempotent.

Reply:
But we have no idea what `fullscreenHook` does so it's only idempotent with the default one.